docs/Feature Documentation/Audio and Video Streaming Setup/index.md

```diff
@@ -170,33 +115,33 @@ gst-launch-1.0 souphttpsrc location=http://127.0.0.1:5050 ! "application/x-rtp-s
 gst-launch-1.0 souphttpsrc location=http://127.0.0.1:5080 ! audio/x-raw,format=S16LE,rate=16000,channels=1 ! pulsesink
 ```
 
-# Start SDL Core
-In the build folder
+## Video Streaming States
 
-```
-cd bin
-./start.sh
-```
-!!! NOTE
-If you want to use a USB connection, you must run
-```
-sudo ./start.sh
-```
-!!!
+This section describes how Core manages the streaming states of mobile applications. Only one application may stream video at a time, but audio applications may stream while in the LIMITED state with other applications.
+
+When an app is moved to HMI level `FULL`:
+- All non-streaming applications go to HMI level `BACKGROUND`
+- All apps with the same App HMI Type go to `BACKGROUND`
+- Streaming apps with a different App HMI Type that were in `FULL` go to `LIMITED`
+
+When an app is moved to HMI level `LIMITED`:
+- All non-streaming applications keep their HMI level
+- All applications with a different App HMI Type keep their HMI level
+- Applications with the same App HMI Type go to `BACKGROUND`
 
-### Start the web HMI
+### Start the Web HMI
 
 CD into the HMI repository and run
 ```
 chromium-browser index.html
 ```
 
-### Start Video or Audio Stream
+### Additional Resources
 
 !!! NOTE
-No open source mobile application currently implements video streaming.
+Livio provides an [example video streaming android application](https://github.com/livio/sdl_video_streaming_android_sample).
 !!!
 
-[iOS Video Streaming Guide](../../iOS/mobile-navigation/video-streaming/)
+[iOS Video Streaming Guide](../../../iOS/video-streaming-for-navigation-apps/video-streaming/)
 
-[Android Video Streaming Guide](../../Android/mobile-navigation/video-streaming/)
+[Android Video Streaming Guide](../../../android/video-streaming-for-navigation-apps/video-streaming/)
```

Comment:
```suggestion
[Android Video Streaming Guide](https://smartdevicelink.com/en/guides/android/video-streaming-for-navigation-apps/video-streaming/)
```

Reply:
The relative links should work once this is merged, any reason that full paths are better?

Reply:
Full paths would be better in this case since this is basically an external resource(not part of sdl_core_guides) and the link would always work regardless of any modifications to our local config structure

Also, we usually only use relative links for local resources. For linking to external guides we use the full path( for example the link to the policy overview https://github.com/smartdevicelink/sdl_core_guides/pull/65/files#diff-4d81113b72a417a09e4e6b1cb2028e32R50)

Reply:
The iOS guide is a local resource, they're both on the same hard drive hosted by the same server on the same domain. Using a full path will break the link on sites like staging. If you're sure the correct trade off is to use a full path let me know and I'll change it.

Reply:
https://github.com/smartdevicelink/sdl_core_guides/pull/71#discussion_r388485301

Reply:
added in https://github.com/smartdevicelink/sdl_core_guides/pull/71/commits/9f48ec382fbbe03a31807656c63517457e3cf268